common/core/gpudevice.cpp

```diff
@@ -201,6 +201,8 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
           "UpdateDisplayState.");
       UpdateDisplayState();
     }
+    /* Adding sleep to avoid busy looping before a retry */
+    usleep(1000);
```

Comment:
hmm we break when there is nothing more to read from fd. How can we ensure we are reading everything from fd without the loop ?